doc/en/fixture.rst

```diff
@@ -34,25 +34,54 @@ both styles, moving incrementally from classic to new style, as you
 prefer.  You can also start out from existing :ref:`unittest.TestCase
 style <unittest.TestCase>` or :ref:`nose based <nosestyle>` projects.
 
-The built-in pytest :ref:`fixtures <fixtures-api>` are, in order of
-introduction:
+:ref:`Fixtures <fixtures-api>` are defined using the
+:ref:`@pytest.fixture <pytest.fixture-api>` decorator, :ref:`described
+below <funcargs>`. Pytest has useful built-in fixtures, listed here
+for reference:
+
+   :ref:`capfd-api`
+        Capture, as text, output to file descriptors ``1`` and ``2``
+
+   :ref:`capfdbinary-api`
+        Capture, as bytes, output to file descriptors ``1`` and ``2``
+
+   :ref:`caplog-api`
+        Control logging and access log entries
```

Comment:
This fixture doesn't control logging per se, just how it's captured. Being completely accurate is a little clumsy but that would be something like `Control log capturing and access captured log entries`